crates/cargo-test-support/src/lib.rs

```diff
@@ -1622,6 +1622,24 @@ pub fn basic_lib_manifest(name: &str) -> String {
     )
 }
 
+pub fn basic_bin_manifest_with_readme(name: &str, readme_filename: &str) -> String {
```

Comment:
FWIW I wouldn't bother making a new constructor function for this, it's fine to just write this inline in each test (the manifest)

Reply:
Ah, I could just inline this for most of the tests - but this comes in handy for the `cargo_read_manifest_default_readme` test, where we are looping over an array of the default file names. Is there a better way to do this while also not needing this constructor function?

Reply:
Eh I just find these constructors weird and super old, most tests aren't written with them. Mind moving this constructor to the readme tests? Seems fine to have a shorthand at least for a few tests!